label_studio/io_storages/localfiles/models.py

```diff
@@ -49,8 +49,15 @@ def get_data(self, key):
             document_root = Path(get_env('LOCAL_FILES_DOCUMENT_ROOT', default='/'))
             relative_path = str(path.relative_to(document_root))
             return {settings.DATA_UNDEFINED_NAME: f'/data/local-files/?d={relative_path}'}
-        with open(path) as f:
-            value = json.load(f)
+
+        try:
+            with open(path, encoding='utf8') as f:
+                value = json.load(f)
+        except (UnicodeDecodeError, json.decoder.JSONDecodeError):
```

Comment:
It has been already added in [a separate PR](https://github.com/heartexlabs/label-studio/pull/801)